packages/strapi-plugin-graphql/hooks/graphql/index.js

```diff
@@ -95,7 +95,18 @@ module.exports = strapi => {
         schemaDef.resolvers = resolvers;
       }
 
-      const apolloServerConfig = _.get(strapi.plugins.graphql, 'config.apolloServerConfig', {});
+      // TODO: Remove these deprecated options in favor of `apolloServerConfig` in the next major version
+      const defaultApolloServerConfig = {
```

Comment:
can you name this `deprecatedApolloServerConfig`

Reply:
Done!